schema/xsdtype.cpp

```diff
@@ -49,6 +49,7 @@ XSDType::~XSDType()
 
 XSDType &XSDType::operator=(const XSDType &other)
 {
+    XmlElement::operator=(other);
```

Comment:
This should move to after the `if()`. No need to do it if self-assignment.

Same thing in `ComplexType::operator=`